Hometask_5/Hometask_5.py

```diff
@@ -0,0 +1,96 @@
+from datetime import datetime
+
+PATH = r'C:/Python/Python-for-DQE/Hometask_5/newsfeed.txt'
```

Comment:
Better to use relative path here. Because if someone else try to run this code on his local machine then he will receive an error "Such path does not exist."